CMakeLists.txt

```diff
@@ -65,6 +76,25 @@ execute_process(COMMAND openassetio-traitgen ${PROJECT_BINARY_DIR}/traits.yml
                 COMMAND_ERROR_IS_FATAL ANY
                 COMMAND_ECHO STDERR)
 
+if (OPENASSETIO_MEDIACREATION_GENERATE_PYTHON)
+    # Generate the python package by running traitgen.
```

Comment:
[fix] Can we elaborate that we're explicitly doing this so that we're using the same traitgen as C++ is - so they're guaranteed to be in sync (as that's our reasoning right now for doing it this way). Otherwise it just reads like we did a bad implementation for no reason.

Reply:
Sure, reworded the latter paragraph and made it the main thrust of the docs : 6a27240637fcb9118e3654aea3d27448d1c65ae0

Reply:
Legend - thank you kindly 🙏